docs/end_to_end_tutorials/chatbots/building_a_chatbot.md

```diff
@@ -48,174 +68,106 @@ for year in years:
 
 ### Setting up Vector Indices for each year
 
-We first setup a vector index for each year. Each vector index allows us 
+We first setup a vector index for each year. Each vector index allows us
 to ask questions about the 10-K filing of a given year.
 
 We build each index and save it to disk.
 
 ```python
-# initialize simple vector indices + global vector index
-service_context = ServiceContext.from_defaults(chunk_size=512)
+# initialize simple vector indices
+from llama_index import VectorStoreIndex, ServiceContext, StorageContext
+
 index_set = {}
+service_context = ServiceContext.from_defaults(chunk_size=512)
 for year in years:
     storage_context = StorageContext.from_defaults()
     cur_index = VectorStoreIndex.from_documents(
-        doc_set[year], 
+        doc_set[year],
         service_context=service_context,
         storage_context=storage_context,
     )
     index_set[year] = cur_index
-    storage_context.persist(persist_dir=f'./storage/{year}')
-
+    storage_context.persist(persist_dir=f"./storage/{year}")
 ```
 
 To load an index from disk, do the following
+
 ```python
 # Load indices from disk
+from llama_index import load_index_from_storage
+
 index_set = {}
 for year in years:
-    storage_context = StorageContext.from_defaults(persist_dir=f'./storage/{year}')
-    cur_index = load_index_from_storage(storage_context=storage_context)
+    storage_context = StorageContext.from_defaults(persist_dir=f"./storage/{year}")
+    cur_index = load_index_from_storage(
+        service_context=service_context, storage_context=storage_context
```

Comment:
Quick note, I think storage context has to come first (it's technically an arg rather than a kwarg)